.gitignore

```diff
@@ -0,0 +1,13 @@
+/vendor
+
+*.swp
+.idea
```

Comment:
These should be in your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)